### PR TITLE
fix: Add right code examples

### PR DESCRIPTION
**Pull request by smb2796**
Still needs some touchups.

### CODE REVIEW

**Review by ryanwolhuter — APPROVED** [review]
nice job, we'll have to start putting you to work on the frontend ;)

File: components/Tabs.tsx [hunk 1]
```diff
@@ -37,86 +37,239 @@ contract OO_GettingStarted {
...
+    },
     {
       title: "Prediction Markets",
       shortTitle: "Predict",
       content: `The OO can validate natural-language statements and answer questions about real-world events.
-      There is a dispute resolution process if something unexpected happens.`,
+      This includes arbitrary questions about weather, sports, elections, markets or anything universally verifiable.`,
       usedBy: "Polymarket",
-      example: "“Did the temperature on the 25th of July 2022 in Manhattan NY exceed 35c?”",
+      example: "“Did the Chiefs beat the Eagles in the 2022-2023?”",
```
Comment (daywiss) [inline components/Tabs.tsx:243]: "in the 2022-2023" doesnt make sense

**Review by daywiss — COMMENTED** [review]
small copy issue

File: components/Tabs.tsx (hunk 1 above)
Comment (smb2796) [reply components/Tabs.tsx]: Good catch. updated

**Pull request merged**